views/partials/_subnav.njk

```diff
@@ -1,6 +1,6 @@
 <nav class="app-subnav">
     {% for item in navigation %}
-      {% if item.url in path %}
+      {% if path.startsWith(item.url) %}
```

Comment:
There's also an issue with the mobile navigation – the 'components' section is open on page load, probably because of exactly the same issue.

Reply:
Ah...